README.md

```diff
@@ -911,6 +911,150 @@ module.exports = {
 };
 ```
 
+### Multiple Themes
+
+**webpack.config.js**
+
+```js
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  entry: "./src/index.js",
+  module: {
+    rules: [
+      {
+        test: /\.s[ac]ss$/i,
+        oneOf: [
+          {
+            resourceQuery: "?dark",
+            use: [
+              Self.loader,
```

Comment:
I don't see a reference to `Self`. Should it be like this?
```suggestion
              MiniCssExtractPlugin.loader,
```
(same for next two occurrences)

Reply:
It is for tests

Reply:
Yes, I saw the tests. But this is the README right?